### PR TITLE
Trying

### PR DESCRIPTION
**Pull request by QZanden**
@halfak, is this a good start?

### CODE REVIEW

File: wikidata.mediawiki [hunk 1]
```diff
@@ -247,41 +247,42 @@ user_actions:
...
-      summary: "%(snuggler)s invited %(user)s to the teahouse"
+      summary: "%(snuggler)s invited %(user)s to the project chat"
       fields:
         - # template
           name: template
           label: Template
           type: radio
           tooltip: the template to be used for the invitation.
-          default: :Wikipedia:Teahouse/Invitation
+          default: :Wikidata:Teahouse/Invitation
```
Comment (halfak) [inline wikidata.mediawiki]: Looks like wikidata doesn't have a Teahouse, so this wouldn't work.  Is there some other newcomer help space we could direct people to?

File: wikidata.mediawiki [hunk 2]
```diff
@@ -310,12 +311,12 @@ user_actions:
       operations:
         - # Appends a list item to the bottom of AIV
           type: append
-          page_name: Wikipedia:Administrator_intervention_against_vandalism
+          page_name: Administrators%27_noticeboard
```
Comment (halfak) [inline wikidata.mediawiki]: This looks wrong.  Should it be `Wikidata:Administrators'_noticeboard`?

Comment (QZanden) [reply wikidata.mediawiki]: I don't know, I took the link that was in the URL, but I don't know what this program accepts...

File: wikidata.mediawiki (hunk 1 above)
Comment (QZanden) [reply wikidata.mediawiki]: That's right, maybe the [[WD:Project chat]]? Although I do not know if that is a friendly place for beginners...

**Comment by QZanden** [comment]
@halfak What's the next step? I guess creating the missing templates?

**Pull request merged**